.fixtures.yml

```diff
@@ -8,11 +8,13 @@ fixtures:
     inifile: https://github.com/puppetlabs/puppetlabs-inifile.git
     mcollective: https://github.com/choria-io/puppet-mcollective.git
     ruby: https://github.com/puppetlabs/puppetlabs-ruby.git
-    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
     systemd: https://github.com/voxpupuli/puppet-systemd.git
     vcsrepo: https://github.com/puppetlabs/puppetlabs-vcsrepo.git
     yumrepo_core: https://github.com/puppetlabs/puppetlabs-yumrepo_core.git
   forge_modules:
+    stdlib:
+      repo: "puppetlabs/stdlib"
+      ref: "8.6.0"
```

Comment:
puppetlabs/stdlib released 9.0.0 which removes a bunch of deprecated functions. This caused issues with tests in files unrelated to this PR. We will need to address those, but until then, we need to pin the stdlib fixture for tests.